src/components/block/FooterCTACard/FooterCTACard.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import FooterCTACard from "./FooterCTACard";
+
+export default {
+  title: "Block/FooterCTACard",
+  component: FooterCTACard,
+};
+
+const Template = ({ subtitle, body, buttonText }) => (
+  <div style={{ backgroundColor: "black", width: "480px" }}>
+    <FooterCTACard subTitle={subtitle} body={body} buttonText={buttonText} />
```

Comment:
```suggestion
    <FooterCTACard subtitle={subtitle} body={body} buttonText={buttonText} />
```

You mistyped the prop name here, which causes the title to not actually show up in the Storybook story.